podman.md

```diff
@@ -16,7 +16,7 @@ These are the credentials used to connect to the dbs:
 
 * Server
 ```
-podman run -it --rm=true --memory-swappiness=0 --name HibernateTestingPGSQL -e POSTGRES_USER=hreact -e POSTGRES_PASSWORD=hreact -e POSTGRES_DB=hreact -p 5432:5432 postgres:12
```

Comment:
why deleting `memory-swappiness` ?

Reply:
rootless Postgres wasn't working with the --memory-swappiness=0 option

Reply:
it works fine for me?

Reply:
OK, but I tested it with Andrea and it doesn't work for him either. Let's hear from @gbadner

Reply:
Let's merge it - but I'd still be curious to know of you all happen to test it further. No need to change back though: if it's problematic it's problematic.